packages/components/src/internal/components/forms/actions.ts

```diff
@@ -427,3 +430,16 @@ export function useUsersWithPermissions(
 
     return { error, loadingState, users };
 }
+
+export function updateRowFieldValue(model: QueryModel, name: string, value: any): Promise<any> {
+    const options: any = {
```

Comment:
nit: Seems better to just declare this within the `updateRows` so we don't escape the typings here.

```ts
return updateRows({
    schemaQuery: model.schemaQuery,
    // ...
});
```